python/tissue:histone_analysis.py

```diff
@@ -0,0 +1,218 @@
+        fibroblast.append(expression_df[i])
+    if tissue_dict[celllines] == "SOFT TISSUE":
+        soft_tissue.append(expression_df[i])
+    if tissue_dict[celllines] == "BILIARY TRACT":
+        biliary_tract.append(expression_df[i])
+    if tissue_dict[celllines] == "AUTONOMIC GANGLIA":
+        autonomic_ganglia.append(expression_df[i])
+    if tissue_dict[celllines] == "PLEURA":
+        pleura.append(expression_df[i])
+    if tissue_dict[celllines] == "URINARY TRACT":
+        urinary_tract.append(expression_df[i])
+    if tissue_dict[celllines] == "KIDNEY":
+        kidney.append(expression_df[i])
+    if tissue_dict[celllines] == "OESOPHAGUS":
+        oesophagus.append(expression_df[i])
+    if tissue_dict[celllines] == "THYROID":
+        thyroid.append(expression_df[i])
+    if tissue_dict[celllines] == "SALIVARY GLAND":
+        salivary_gland.append(expression_df[i])
+    i = i + 1
+   
+#Convert to numpy arrays     
+lung = np.array(lung)
+ovary = np.array(ovary)
+large_intestine = np.array(large_intestine)
+central_nervous_system = np.array(central_nervous_system)
+haematopoietic_and_lymphoid_tissue =np.array(haematopoietic_and_lymphoid_tissue)
+pancreas = np.array(pancreas)
+upper_aerodigestive_tract = np.array(upper_aerodigestive_tract)
+breast = np.array(breast)
+prostate =np.array(prostate)
+stomach = np.array(stomach)
+endometrium = np.array(endometrium)
+bone = np.array(bone)
+skin =np.array(skin)
+liver = np.array(liver)
+fibroblast = np.array(fibroblast)
+soft_tissue = np.array(soft_tissue)
+biliary_tract = np.array(biliary_tract)
+autonomic_ganglia = np.array(autonomic_ganglia)
+pleura = np.array(pleura)
+urinary_tract = np.array(urinary_tract)
+kidney = np.array(kidney)
+oesophagus = np.array(oesophagus)
+thyroid = np.array(thyroid)
+salivary_gland = np.array(salivary_gland)
+ 
+#Average
+lung = np.mean(lung,axis = 0)
+ovary = np.mean(ovary,axis = 0)
+large_intestine = np.mean(large_intestine,axis = 0)
+central_nervous_system = np.mean(central_nervous_system,axis = 0)
+haematopoietic_and_lymphoid_tissue = np.mean(haematopoietic_and_lymphoid_tissue,axis = 0)
+pancreas = np.mean(pancreas,axis = 0)
+upper_aerodigestive_tract = np.mean(upper_aerodigestive_tract,axis = 0)
+breast = np.mean(breast,axis = 0)
+prostate = np.mean(prostate,axis = 0)
+stomach = np.mean(stomach,axis = 0)
+endometrium = np.mean(endometrium,axis = 0)
+bone = np.mean(bone,axis = 0)
+skin = np.mean(skin,axis = 0)
+liver = np.mean(liver,axis = 0)
+fibroblast = np.mean(fibroblast,axis = 0)
+soft_tissue = np.mean(soft_tissue,axis = 0)
+biliary_tract = np.mean(biliary_tract,axis = 0)
+autonomic_ganglia = np.mean(autonomic_ganglia,axis = 0)
+pleura = np.mean(pleura,axis = 0)
+urinary_tract = np.mean(urinary_tract,axis = 0)
+kidney = np.mean(kidney,axis = 0)
+oesophagus = np.mean(oesophagus,axis = 0)
+thyroid = np.mean(thyroid,axis = 0)
+salivary_gland = np.mean(salivary_gland,axis = 0)  
+
+"""
+HISTONE ANALYSI
+"""
+h3k4 = df.iloc[:,3:7]
```

Comment:
Need to somehow automate this process. Rather than looking at specific locations (using df.iloc), you may want to name your columns by their histone markers and search based on that.